setup.py

```diff
@@ -19,7 +19,8 @@
     "multiprocessing-logging",
     "psutil",
     "configobj",
-    "tensorflow>=2.5.0",
+    'tensorflow>=2.5.0;  platform_system!="Darwin" or platform_machine!="arm64"',
+    'tensorflow-macos>=2.5.0;  platform_system=="Darwin" and platform_machine=="arm64"',
```

Comment:
Should this be just:
`'tensorflow-macos>=2.5.0;  platform_system=="Darwin"` ? since x86-64 is also supported by tensorflow-macos on pypi and seems to be the way to get metal acceleration under Intel+AMD?
Basically, the logic should let a x86-64 macOS have *either* `tensorflow` or `tensorflow-macos`. But M1 has to be (for the time being anyways) the latter.

Reply:
Or will that change make every mac have to use apple's TF-macos...

Reply:
I dug out my old laptop and gave this a go
<img width="725" alt="image" src="https://user-images.githubusercontent.com/15052188/151186642-b1525078-33c6-4ac2-9e25-9449c7556854.png">